README.rst

```diff
@@ -68,3 +70,21 @@ containing the kwargs for colormaps.create() like this::
           [1.000, [255, 255, 255, 255]]
       ]
     }
+
+
+Development installation
+------------------------
+
+For development, you can use a docker-compose setup::
+
+    $ docker-compose build --build-arg uid=`id -u` --build-arg gid=`id -g` lib
+    $ docker-compose up --no-start
+    $ docker-compose start
+    $ docker-compose exec lib bash
+
+    (docker)$ virtualenv . --python python3
```

Comment:
Werkt dat? Volgens de help is het: `-p PYTHON_EXE, --python=PYTHON_EXE`

Reply:
Het werkt! Maar je hebt gelijk, ik zal me aan de calling syntax houden.